pkg/portfwd/client.go

```diff
@@ -130,14 +94,32 @@ func (g GrpcClientRW) Write(p []byte) (n int, err error) {
 func (g GrpcClientRW) Read(p []byte) (n int, err error) {
 	in, err := g.stream.Recv()
 	if err != nil {
-		if errors.Is(err, io.EOF) {
-			return 0, nil
-		}
 		return 0, err
```

Comment:
If we don't have proper logging in tcpproxy, we can mitigate it here by logging real read errors. Same can be done for Write().

Reply:
Resolving as we are using bicopy